src/tips-and-information/TipsAndInformation.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useEffect, useState} from 'react';
+import {getTextForLanguage, useTranslation} from '@atb/translations';
+import {ThemeText} from '@atb/components/text';
+import firestore from '@react-native-firebase/firestore';
+import {TipRaw, TipType} from '@atb/tips-and-information/types';
+import {mapToTips} from '@atb/tips-and-information/converters';
+import {ExpandableSectionItem, Section} from '@atb/components/sections';
+import {sortTipsByTitle} from '@atb/tips-and-information/sort-tips-by-title';
+
+export const TipsAndInformation = () => {
+  const {language} = useTranslation();
+  const [currentlyOpen, setCurrentlyOpen] = useState<number>();
+
+  const [tips, setTips] = useState<TipType[]>([]);
+
+  useEffect(
+    () =>
+      firestore()
+        .collection<TipRaw>('tipsAndInformation')
+        .onSnapshot(
+          async (snapshot) => {
+            const newTips = mapToTips(snapshot.docs);
+            const sortedTips = sortTipsByTitle(newTips, language);
+            setTips(sortedTips);
+          },
+          (err) => {
+            console.warn(err);
+          },
+        ),
+    /*
+    Resubscribing on language change? Doh. Note that we shouldn't have
```

Comment:
Is this comment meant to be here?

Reply:
This is not code that's written for this pr. It's a file that's moved, but Git couldn't keep up through various rebases etc. The original code was written by the students.

Reply:
Ahh, okey

Reply:
Made a fix anyways: https://github.com/AtB-AS/mittatb-app/pull/4142/commits/94e7f9e8891a7239ce3260f191d93802c5afd5e7